src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -77,8 +77,8 @@ private void registerGATKClasses(Kryo kryo) {
         kryo.register(SAMFileHeader.SortOrder.class);
         kryo.register(SAMProgramRecord.class);
         kryo.register(SAMReadGroupRecord.class);
-
-        //register to avoid writing the full name of this class over and over
-        kryo.register(PairedEnds.class, new FieldSerializer<>(kryo, PairedEnds.class));
+        kryo.register(PairedEnds.class, new PairedEnds.PairedEndsEmptyFragmentSerializer());
```

Comment:
This seems wrong, it should be registering for each of the subclasses, not the abstract class